frontend/src/components/Admin_Dashboard/admin-dashboard.js

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import "./admin-dashboard.css";
+import {
+  Card,
+  CardHeader,
+  ImageHeader,
+  CardBody,
+  CardFooter,
+} from "react-simple-card";
+const Dashboard = () => {
+  return (
+    <div>
+      <div className="welcome-section">
+        <h1>Good Morning, Admin</h1>
+        <div className="welcome-section-activity">
+          <div className="row">
+            <div className="col-md-10">
+              <h3 className="online">
+                {" "}
+                <i class="fas fa-circle"></i>12 Online
+              </h3>
+              <h3 className="offline">
+                <i class="far fa-circle"></i>12 Offline
+              </h3>
+            </div>
+            <div className="col-md-2 "></div>
+          </div>
+        </div>
+      </div>
+      <div className="team container">
+        <h1>Dashboard</h1>
+        <div className="dash"></div>
+        <div className="container">
+          <div className="row justify-content-center">
+            <div className="col-lg-4 col-md-6 col-sm-8 my-3">
+              <div
+                className="single-features d-flex mt-30 wow fadeIn"
+                data-wow-duration="1s"
+                data-wow-delay="0s"
+              >
+                <div className="features-icon">
+                  <i class="fa fa-bullhorn" aria-hidden="true"></i>
+                </div>
+                <div className="features-content text-left">
+                  <h4 className="features-title">
+                    <a href="/#">Broadcasts</a>
+                  </h4>
+                  <p className="text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit,
```

Comment:
Hey @Utkarshn10 let's replace this with the count (put dummy value here as of now)
Also please make the card size small and and the icon size should also be a bit smaller.

Reply:
> Hey @Utkarshn10 let's replace this with the count (put dummy value here as of now)
> Also please make the card size small and and the icon size should also be a bit smaller.

@Kajol-Kumari Changes done. Icons size LGTM  as it looks cleaner. 
![Screenshot (15)](https://user-images.githubusercontent.com/58587256/104172343-228da580-542a-11eb-8e50-269c973c9438.png)

Reply:
Also @Kajol-Kumari I am having this issue in commiting the final change that I just send Screenshot of. Could you please help.
![Screenshot (16)](https://user-images.githubusercontent.com/58587256/104183258-b87bfd00-5437-11eb-8927-76d891f36e1e.png)

Reply:
Hey @Utkarshn10 seems like you are getting lint errors. Just read the error properly it will point you to the place where you need to format the code. Please fix that and it will be good to go. In cade you still face issue, ping me on slack :)